faust/fixups/django.py

```diff
@@ -63,22 +63,28 @@ def enabled(self) -> bool:
                 return True
         return False
 
+    def wait_for_django(self) -> None:
+        import django
+
+        django.setup()
+        if self.settings.DEBUG:
```

Comment:
Will this warn twice?

Reply:
Ah yes good catch. Could wrap this in functools.lru_cache? You only need to wait for this once, but it will make testing a bit trickier